### PR TITLE
SJQ2T-324: BE Permissions For Edit Bucket

### PR DESCRIPTION
**Pull request by DarkoVeljanoskiHtec**

### CODE REVIEW

File: backend/src/main/java/com/htecgroup/skynest/service/impl/BucketServiceImpl.java [hunk 1]
```diff
@@ -151,6 +148,7 @@ public void restoreBucket(UUID uuid) {
   @Override
   public BucketResponse editBucket(
       BucketEditRequest bucketEditRequest, @Valid @CurrentUserCanEditBucket UUID uuid) {
+    permissionService.currentUserHasPermissionForBucket(uuid, AccessType.EDIT);
```
Comment (DraganMitrasinovicHtec) [inline backend/src/main/java/com/htecgroup/skynest/service/impl/BucketServiceImpl.java]: Did you test this, I think this annotation is old and it only checks for ownership

Comment (ivanfhtec) [reply backend/src/main/java/com/htecgroup/skynest/service/impl/BucketServiceImpl.java]: the method is old, but should work

Comment (ivanfhtec) [reply backend/src/main/java/com/htecgroup/skynest/service/impl/BucketServiceImpl.java]: not sure the annotation is necessary

Comment (DarkoVeljanoskiHtec) [reply backend/src/main/java/com/htecgroup/skynest/service/impl/BucketServiceImpl.java]: should i remove it ?

Comment (DraganMitrasinovicHtec) [reply backend/src/main/java/com/htecgroup/skynest/service/impl/BucketServiceImpl.java]: Yeah, just remove the annotation

**Review by ivanfhtec — APPROVED** [review]

**Pull request merged**